android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -829,8 +829,7 @@ private String makeFontPaths(String font) {
         if (sourceArray != null) {
           for (int i = 0; i < sourceArray.length(); i++) {
             fontFile = sourceArray.getString(i);
-            if (fontFile.contains(".ttf") ||
-              (fontFile.contains(".svg") && (Build.VERSION.SDK_INT == Build.VERSION_CODES.JELLY_BEAN_MR1 || Build.VERSION.SDK_INT == Build.VERSION_CODES.JELLY_BEAN_MR2))) {
+            if (FileUtils.hasFontExtension(fontFile)) {
```

Comment:
This removes support for .svg fonts. I think that's what we want but just checking.

Reply:
I've reverted .svg support for those versions of SDK.